src/main/java/uk/gov/hmcts/reform/bulkscan/payment/processor/service/servicebus/handler/MessageProcessingResultType.java

```diff
@@ -0,0 +1,7 @@
+package uk.gov.hmcts.reform.bulkscan.payment.processor.service.servicebus.handler;
+
+public enum MessageProcessingResultType {
+    SUCCESS,
+    UNRECOVERABLE_FAILURE,
+    POTENTIALLY_RECOVERABLE_FAILURE
+}
```

Comment:
This looks like a copy/paste from another project. Perhaps we won't even need similar approach, perhaps development can be done individually in some way. Especially when idempotency is guaranteed from day one unlike the other project

Reply:
for now 
SUCCESS . = success no retry , http status 200,201
UNRECOVERABLE_FAILURE = failure no retry, http 409
POTENTIALLY_RECOVERABLE_FAILURE  = failure retry ,other http status codes